src/python/espressomd/visualization_opengl.pyx

```diff
@@ -196,24 +199,35 @@ class openGLLive(object):
             'cyanplastic':  [[0, 0.1, 0.06], [0, 0.50980392, 0.50980392], [0.50196078, 0.50196078, 0.50196078], 0.25],
             'greenplastic': [[0, 0, 0], [0.1, 0.35, 0.1], [0.45, 0.55, 0.45], 0.25],
             'redplastic':   [[0, 0, 0], [0.5, 0, 0], [0.7, 0.6, 0.6], 0.25],
+            'blueplastic':  [[0, 0, 0], [0.0, 0.0, 0.5], [0.6, 0.6, 0.7], 0.25],
             'whiteplastic': [[0, 0, 0], [0.55, 0.55, 0.55], [0.7, 0.7, 0.7], 0.25],
             'yellowplastic': [[0, 0, 0], [0.5, 0.5, 0], [0.6, 0.6, 0.5], 0.25]}
 
         # DEFAULT PROPERTIES
         self.specs = {
             'window_size': [800, 800],
             'name': 'Espresso Visualization',
+
             'background_color': [0, 0, 0],
-            'update_fps': 30.0,
-            'periodic_images': [0, 0, 0],
+
+            'draw_fps': False,
             'draw_box': True,
             'draw_axis': True,
+            'draw_nodes': False,
+            'draw_cells': False,
+
```

Comment:
Maybe remove blank lines and divide your specs into subspecs to maintain a logical structure

Reply:
I'd rather leave them in. With subspecs, the used would need to know which dict to address.